Submodule externals/nx_tzdb/tzdb_to_nx contains modified content
diff --git a/externals/nx_tzdb/tzdb_to_nx/src/tzdb/CMakeLists.txt b/externals/nx_tzdb/tzdb_to_nx/src/tzdb/CMakeLists.txt
index 2be93dc..c330d58 100644
--- a/externals/nx_tzdb/tzdb_to_nx/src/tzdb/CMakeLists.txt
+++ b/externals/nx_tzdb/tzdb_to_nx/src/tzdb/CMakeLists.txt
@@ -13,23 +13,23 @@ set(NX_ZONEINFO_DIR "${NX_TZDB_DIR}/zoneinfo")
 
 set(TZDB_VERSION_FILE ${TZ_SOURCE_DIR}/NEWS)
 
-execute_process(
-    COMMAND 
-        ${GIT_PROGRAM} log --pretty=%at -n1 NEWS
-    OUTPUT_VARIABLE
-        TZ_COMMIT_TIME
-    WORKING_DIRECTORY
-        ${TZ_SOURCE_DIR}
-    COMMAND_ERROR_IS_FATAL ANY)
+#execute_process(
+#    COMMAND 
+#        ${GIT_PROGRAM} log --pretty=%at -n1 NEWS
+#    OUTPUT_VARIABLE
+#        TZ_COMMIT_TIME
+#    WORKING_DIRECTORY
+#        ${TZ_SOURCE_DIR}
+#    COMMAND_ERROR_IS_FATAL ANY)
 
-string(REPLACE "\n" "" TZ_COMMIT_TIME "${TZ_COMMIT_TIME}")
+#string(REPLACE "\n" "" TZ_COMMIT_TIME "${TZ_COMMIT_TIME}")
 
-execute_process(
-    COMMAND
-        ${GNU_DATE} +%y%m%d --date=@${TZ_COMMIT_TIME}
-    OUTPUT_VARIABLE
-        TZDB_VERSION
-    COMMAND_ERROR_IS_FATAL ANY)
+#execute_process(
+#    COMMAND
+#        ${GNU_DATE} +%y%m%d --date=@${TZ_COMMIT_TIME}
+#    OUTPUT_VARIABLE
+#        TZDB_VERSION
+#    COMMAND_ERROR_IS_FATAL ANY)
 
 set(NX_VERSION_FILE ${NX_TZDB_DIR}/version.txt)
 file(WRITE ${NX_VERSION_FILE} "${TZDB_VERSION}")
